src/databricks/labs/ucx/install.py

```diff
@@ -415,8 +440,7 @@ def config(self):
     def folder(self):
         return self._installation.install_folder()
 
-    def run(self):
-        logger.info(f"Installing UCX v{self._product_info.version()}")
+    def run(self, silent: bool = False, trigger_job: bool = False):
```

Comment:
```suggestion
    def run(self, trigger_job: bool = False):
```
If we have accounts install, why do we need silent?